test_runner/regress/test_pgstat.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from fixtures.neon_fixtures import AuxFileStore, NeonEnv
+from fixtures.pg_version import PgVersion
+
+
+#
+# Test that pgstat statistic is preserved across sessions
+#
+@pytest.mark.parametrize(
+    "pageserver_aux_file_policy", [AuxFileStore.V1, AuxFileStore.V2, AuxFileStore.CrossValidation]
+)
+def test_pgstat(neon_simple_env: NeonEnv):
+    env = neon_simple_env
+    if env.pg_version == PgVersion.V14:
+        pytest.skip("PG14 doesn't support pgstat statistic persistence")
+
+    n = 10000
+    env.neon_cli.create_branch("test_pgstat", "empty")
+    endpoint = env.endpoints.create_start("test_pgstat")
```

Comment:
```suggestion
    endpoint = env.endpoints.create_start("main")
```
no need to create a branch